### PR TITLE
tool to rotate normal maps

### PR DESCRIPTION
**Pull request by kerautret**
# PR Description
New tool to apply a rotation in the input normal map (it rotates both the position and normals orientation to be consistant.

# Checklist

- [x] Doxygen documentation of the code completed (classes, methods, types, members...)
- [x] Check if it follows the tools structure described in [CONTRIBUTING.md](https://github.com/DGtal-team/DGtalTools-contrib/blob/master/CONTRIBUTING.md)
- [x] New entry in the [ChangeLog.md](https://github.com/DGtal-team/DGtalTools-contrib/blob/master/ChangeLog.md) added.
- [x] Update the readme with potentially a screenshot of the tools if it applies. 
- [x] No warning raised in Debug ```cmake``` mode (otherwise, Travis C.I. will fail).
- [x] All continuous integration tests pass (Github Actions & appveyor).

### CODE REVIEW

File: ChangeLog.md [hunk 1]
```diff
@@ -1,6 +1,10 @@
 # DGtalTools-contrib  1.4 (beta)
 
-
+- *Geometry2d*
+  - rotNmap: new tools tp apply a rotation in the input normal map (it
```
Comment (dcoeurjo) [inline ChangeLog.md:4]:
```suggestion
  - rotNmap: new tools to apply a rotation in the input normal map (it
```

File: README.md [hunk 2]
```diff
@@ -49,7 +49,7 @@ This section can contains various utilities related to 2d geometry (2d estimator
       - CumulativeSequenceTest and RationalBeattySequenceTest: tests from
         LUTBasedNSDistanceTransform.
    - rosinThreshold : applies the Rosin Threshold algorithm on a simple file of data values.
-   
+   - rotNmap: Applies a rotation in the input normal map (it rotates both the position and normals orientation to be consistant.
```
Comment (dcoeurjo) [inline README.md:52]:
```suggestion
   - rotNmap: Applies a rotation in the input normal map (it rotates both the position and normals orientation to be consistant).
```

**Review by dcoeurjo — APPROVED** [review]
Few suggestions. thx

File: ChangeLog.md (hunk 1 above)
Comment (kerautret) [reply ChangeLog.md:4]: thanks

**Comment by kerautret** [comment]
thanks for the review, merging ;)

**Pull request merged**